src/meta/protos/proto/file_format.proto

```diff
@@ -104,6 +104,9 @@ message CsvFileFormatParams {
   string escape = 6;
   string quote = 7;
   string null_display = 8;
+  // corresponding to `!error_on_column_count_mismatch`
+  // for we can not set default value in proto3
+  bool allow_column_count_mismatch = 9;
```

Comment:
Seems the boolean default value in protobuf3 is false.
https://protobuf.dev/programming-guides/proto3/#default

Reply:
yes, that is why I do not use error_on_column_count_mismatch here